config/qualifications.yml

```diff
@@ -40,20 +44,70 @@ description: >
   - In section 1.5.1, there are instructions to set up a Heroku account to deploy your code to.
     Since you'll be deploying code to production in a few short months, you can skip all related steps.
 ---
-title: 'Complete a ruby tutorial'
+title: Complete a ruby tutorial
 description: >
   There are many Ruby tutorials available which you can choose from.
   Pick one from this list, or if you find one you like better, get approval from your supervisor.
   - [Learn enough Ruby to be dangerous](https://www.learnenough.com/ruby-tutorial/hello_world)
   - [Ruby Koans](http://rubykoans.com/)
   rubymonk(maybe), learn enough Ruby to be dangerous, ruby koans
-title: 'Complete a git tutorial (optional)'
+---
+title: Complete a git tutorial (optional)
 description: >
   If you can push, pull, merge, and resolve conflics with git, you're all set.
   Otherwise, consider a tutorial [like this](https://www.learnenough.com/git-tutorial/getting_started)
-title: 'Complete a command line tutorial (optional)'
+---
+title: Complete a command line tutorial (optional)
 description: >
   Having a good grasp of the command line is essential for working in our team,
```

Comment:
If it's essential why is it optional?

Reply:
If someone already has a good grasp of the command line, they don't have to complete a tutorial. Maybe I can state somewhere "if you can already do this, just check it off" but that's kind of true of most of the items.